_includes/test-infrastructure.html

```diff
@@ -0,0 +1,18 @@
+{% assign section = site.data.test-infrastructure %}
+
+<section class="test-infrastructure site-section" id="test-infrastructure">
+  <div class="container">
+    <div class="row">
+
+      <div class="col-12 col-md-6 col-lg-5 offset-lg-1 text-center">
+        <img class="test-infrastructure-image img-fluid" src="/images/{{ section.image }}" alt="{{ section.title }}">
```

Comment:
We should be using `srcset` or `picture` tag to load retina ready images if it is needed (No worries for IE support)